docs/data-cookbooks/queue-post-to-cmr.md

```diff
@@ -0,0 +1,128 @@
+---
+id: queue-post-to-cmr
+title: Queue PostToCmr
+hide_title: false
+---
+
+In this document, we will walktrough handling CMR errors in workflows by queueing PostToCmr. We will assume that the user already has an ingest workflow setup.
+
+## Overview
+
+The general concept is that the last task of the ingest workflow will be `QueueWorkflow`, which queues the publish workflow. The publish workflow is a workflow containing the `PostToCmr` task. When there is a CMR error from `PostToCmr`, the publish workflow will add itself back onto the queue so that it can be executed when CMR is back online. This is achieved by leveraging the `QueueWorkflow` task again in the publish workflow. The following diagram demonstrates this queueing process.
+
+![Diagram of workflow queueing](../assets/queue-workflow.png)
+
+## Ingest Workflow
+
+The last step should be the `QueueWorkflow` step. It should be configured with a queueUrl and workflow. In this case, the queueUrl is a [throttled queue](../throttling-queued-executions). You can pass any `queueUrl` here. for example, if you would like to specify a lower priority queue. The workflow is the unprefixed workflow name that you would like to queue (e.g. `PublishWorkflow`).
```

Comment:
"for example" -> "For example"
"for example, if you would like to specify a lower priority queue." is not a finished sentence.